crates/e2e/src/subxt_client.rs

```diff
@@ -46,14 +46,24 @@ use ink_env::{
     },
```

Comment:
I am personally not a fun of renaming the file into `subxt_client.rs`. IMO, it is better either to keep the name `client.rs` or rename it to the `impls.rs` like it's done in other ink! crates. Since, all the create provides an implementation for the backend traits

Reply:
I was intending to put drink backend into a separate module (like `drink_client.rs`) to keep clear isolation of the very different code and hence this renaming. Would you suggest putting everything into a signle `client.rs`/`impls.rs` module?

Reply:
If you plan to integrate drink backend in the ink! e2e crate directly, then it makes sense to keep `subxt_client.rs` name

Reply:
that's the plan, so that a developer when writing tests can only specify which backend they need without worrying about dependencies or imports

Reply:
Then it makes sense to keep the name